vendor/knative.dev/reconciler-test/pkg/images/ko.go

```diff
@@ -18,12 +18,17 @@ package images
 
 import (
 	"fmt"
+	"os"
 )
 
 // Use ko to publish the image.
 func KoPublish(path string) (string, error) {
-	fmt.Println(fmt.Sprintf("ko publish -B %s", path))
-	out, err := runCmd(fmt.Sprintf("ko publish -B %s", path))
+	platform := os.Getenv("PLATFORM")
+	if len(platform) > 0 {
+		platform = "--platform=" + platform
+	}
+	fmt.Println(fmt.Sprintf("ko publish %s -B %s", platform, path))
```

Comment:
remove Println

Reply:
Thanks for the comment. You meant to use `Printf` in short, right?